packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-global/typed-array-static/output.mjs

```diff
@@ -1,7 +1,6 @@
 import "core-js/modules/es.array.iterator.js";
 import "core-js/modules/es.array-buffer.slice.js";
 import "core-js/modules/es.object.to-string.js";
-import "core-js/modules/es.typed-array.int8-array.js";
```

Comment:
@zloirock Does `Int8Array.of();` need this polyfill?

Reply:
Yes, since in some cases required fixing or complete replacement of typed arrays.

Reply:
I updated the deps list to add it back.